demo-app/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es6",
+    "target": "ES2018",
```

Comment:
Had to change the target to fix a warning about capture groups not working on earlier versions here: https://github.com/open-truss/open-truss/blob/2277e1a39e73079836bf9d97ce74a791bff6e493/demo-app/src/lib/kusto-uqi-client.ts#L36